aiohttp/web_fileresponse.py

```diff
@@ -142,7 +142,9 @@ def _get_file_path_stat_and_gzip(
 
     async def prepare(self, request: "BaseRequest") -> Optional[AbstractStreamWriter]:
         loop = asyncio.get_event_loop()
-        check_for_gzipped_file = "gzip" in request.headers.get(hdrs.ACCEPT_ENCODING, "")
+        check_for_gzipped_file = (
```

Comment:
Would you please add a comment per https://github.com/aio-libs/aiohttp/pull/8063#discussion_r1468463470